src/components/AppFooter/AppFooter.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import React, { SetStateAction } from 'react';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  amountOfActive: number,
+  completedTodosLength: number,
+  statusFilter: Filter,
+  onChangeStatusFilter: React.Dispatch<SetStateAction<Filter>>,
+  clearCompletedTodos: () => void,
+};
+
+export const AppFooter: React.FC<Props> = ({
+  amountOfActive,
+  completedTodosLength,
+  statusFilter: status,
+  onChangeStatusFilter,
+  clearCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${amountOfActive} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={classNames('filter__link',
+            { selected: status === 'All' })}
+          onClick={() => onChangeStatusFilter('All')}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={classNames('filter__link',
+            { selected: status === 'Active' })}
+          onClick={() => onChangeStatusFilter('Active')}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={classNames('filter__link',
+            { selected: status === 'Completed' })}
+          onClick={() => onChangeStatusFilter('Completed')}
+        >
+          Completed
+        </a>
+      </nav>
+
+      <button
+        data-cy="ClearCompletedButton"
+        type="button"
+        className="todoapp__clear-completed"
+        onClick={() => {
+          onChangeStatusFilter('All');
+          clearCompletedTodos();
+          // eslint-disable-next-line no-console
+          // console.log('click');
+        }}
+        hidden={completedTodosLength === 0}
```

Comment:
You may use this option below in order to hide "Clear button" and keep your filter in center of the footer 
> style={{
          visibility: !completedTodosLength ? 'hidden' : 'visible',
        }}
<img width="515" alt="image" src="https://user-images.githubusercontent.com/100049469/215328011-8de6e0cf-ce4b-46e8-98ac-dcf9f628d284.png">